apps/examples/bluetooth/bt_unit_test.c

```diff
@@ -21,10 +21,15 @@
 #include <string.h>
 #include <debug.h>
 
-#include <tinyara/bluetooth/bluetooth.h>
+//#include <tinyara/bluetooth/bluetooth.h>
+#include "bt_unit_test.h"
+#include "bluetooth.h"
 
 int bt_unit_test_main(int argc, char **argv)
 {
-	printf("Enter bt_unit_test_main()\n");
+	PRT("Enter\n");
```

Comment:
BT_PRT

Reply:
I removed the logging code in this PR and modify the title and commit message.
Please review.